src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -190,11 +192,18 @@ public void start() throws IOException {
             // clear panels
             narrativePanel.setTextArea("");
             artPanel.setTextArea("");
+            statsPanel.setTextArea("");
             Room playerCurrentRoom = player.getCurrentRoom();
             narrativePanel.appendTextArea("YOU ARE IN: " + playerCurrentRoom.getRoomName() + "\n");
             narrativePanel.appendTextArea(playerCurrentRoom.getDesc());
             narrativePanel.appendTextArea("ITEMS IN ROOM: " + playerCurrentRoom.getItems() + "\n");
             narrativePanel.appendTextArea("PEOPLE IN ROOM: " + playerCurrentRoom.getNpcs() + "\n");
+            statsPanel.appendTextArea("HEALTH: " + player.getHealth());
+            if (playerCurrentRoom.getRoomMonster() != null) {
```

Comment:
this if condition already exists. you can move your logic inside the monster condition below